cinema/views.py

```diff
@@ -56,3 +76,58 @@
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+    def get_queryset(self):
+        queryset = self.queryset
+        movie_id = self.request.query_params.get("movie")
+        date = self.request.query_params.get("date")
+        if movie_id:
+            movie_id = int(movie_id)
```

Comment:
Ensure that the conversion of `movie_id` to an integer in `get_queryset` handles potential ValueErrors gracefully.